yaml/private/internal.nim

```diff
@@ -5,16 +5,24 @@
 #    distribution, for details about the copyright.
 
 template internalError*(s: string) =
+  # Note: to get the internal stacktrace that caused the error
+  # compile with the `d:debug` flag.
   when not defined(release):
     let ii = instantiationInfo()
     echo "[NimYAML] Error in file ", ii.filename, " at line ", ii.line, ":"
     echo s
     when not defined(JS):
       echo "[NimYAML] Stacktrace:"
-      try: writeStackTrace()
+      try:
+        writeStackTrace()
+        echo "Internal stacktrace:"
```

Comment:
Okay, last change I request: This `echo` should be inside the `if` branch below. Doesn't make sense without actually having a parent stacktrace displayed.

Reply:
Yeah, good point.